library/common/jni/jni_interface.cc

```diff
@@ -1031,3 +1032,121 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_setPreferredNetwork(JNIEnv* env
   return set_preferred_network(static_cast<envoy_engine_t>(engine),
                                static_cast<envoy_network_t>(network));
 }
+
+bool Java_AndroidCertVerifyResult_isIssuedByKnownRoot(JNIEnv* env, jobject result) {
```

Comment:
Can you help me understand how these methods or called? I guess there is some sort of mapping from the c++ function name to some sort of Java class name? Or maybe not?